src/components/header.js

```diff
@@ -0,0 +1,19 @@
+import { AppBar } from "material-ui";
+import colors from "../styles";
+
+const Header = ({children, style}) => {
+	return (
+		<AppBar 
+			style={style}
+			title="Social Tables Engineering"
+			showMenuIconButton={false}
+		/>
+	);
+}
+
+Header.defaultProps = {
+	style: {
+		backgroundColor: colors.pink
+	}
+};
```

Comment:
should you set the propTypes here too? to say colors and colors.pink are required?